docs/01-overview/main-areas/eventing/evnt-01-overview.md

```diff
@@ -2,6 +2,6 @@
 title: Overview
 ---
 
-Eventing in Kyma is used to send and receive events from applications. For example, you can subscribe to events from an external application, and when the user performs an action there, you can trigger your Function or microservice. To subscribe to events, you need to use the Kyma [Subscription CRD](../../../05-technical-reference/06-custom-resources/evnt-01-subscription.md).
+Eventing in Kyma is used to send and receive events from applications. For example, you can subscribe to events from an external application, and when the user performs an action there, you can trigger your Function or microservice. To subscribe to events, you need to use the Kyma [Subscription CRD](../../../05-technical-reference/00-custom-resources/evnt-01-subscription.md).
```

Comment:
```suggestion
Eventing in Kyma is used to send and receive events from applications. For example, you can subscribe to events from an external application, and when the user performs an action there, you can trigger your Function or microservice. To subscribe to events, you need to use the Kyma [Subscription CR](../../../05-technical-reference/00-custom-resources/evnt-01-subscription.md).
```
You create a Subcription CR to subscribe to events, and the Subcription CRD describes what this CR looks like and it should already be preinstalled in Kyma.
Am I right here with the suggestion, @alexandra-simeonova?